ocs_ci/ocs/cluster.py

```diff
@@ -682,6 +682,22 @@ def get_ceph_capacity(self):
 
         return usable_capacity
 
+    def get_ceph_total_capacity(self):
```

Comment:
isn't it similar to https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py#L665?

Reply:
The one you mentioned is for some reason dividing the total capacity into replica, I just dropped the replica so it will have the real total capacity. This has cause all the failures because when it got back to the function that called it, it multiplied it be osd count for some reason, so I thought why divide and then get again the replica (need to create again a cephcluster object), so just copied the function and dropped the /replica divide to simplify the testcase.

Reply:
can't we use the existing function and add a condition for not deciding by replica if not needed?

Reply:
We can, it just that it is used many times and I didn't want to harm any other testcase. I'll do it.

Reply:
Fixed.